src/Console/Command/AbstractCommand.php

```diff
@@ -28,10 +28,11 @@
  */
 abstract class AbstractCommand extends Command
 {
-    const GITHUB_GROUP = 'sonata-project';
-    const GITHUB_USER = 'SonataCI';
-    const GITHUB_EMAIL = 'thomas+ci@sonata-project.org';
-    const PACKAGIST_GROUP = 'sonata-project';
+    const GITHUB_GROUP = 'symfony-cmf';
+    const GITHUB_USER = 'electricmaxxx';
```

Comment:
would we have to create a specific user for the dev-kit?

Reply:
Yes, and i would like to put those parameters in a `.env` (with a checked in `.env.dist` to keep this application reusable and maybe push something back to sonata.